src/binom.rs

```diff
@@ -0,0 +1,34 @@
+// n choose k for a prime modulo
+pub struct Binom {
+    inv: Vec<u64>,
+    fact: Vec<u64>,
+    inv_fact: Vec<u64>,
+    modulo: u64,
+}
+impl Binom {
+    pub fn new(modulo: u64) -> Binom {
+        Binom {
+            inv: vec![1; 2],
+            fact: vec![1; 2],
+            inv_fact: vec![1; 2],
+            modulo,
```

Comment:
compile-time-known-const mod be faster?

Reply:
imo we shud use some mod int cuz its online contests

Reply:
agree

once we add one we can change this

Reply:
I don't think we can use the same "hacks" to change that... so it wouldn't work with the test AFAIK

Reply:
oh trueee, we did use some hacks in c++

Reply:
I'll resolve for now, I think this is good enough hackpack code which is easy enough to change to a `const` mod for the time being